content/kubermatic/master/tutorials_howtos/administration/admin_panel/_index.en.md

```diff
@@ -4,42 +4,106 @@ date = 2018-08-09T12:07:15+02:00
 weight = 40
 +++
 
-The Admin Panel is a place for the Kubermatic administrators where they can manage the global settings that directly
-impact all Kubermatic users. Admin rights can be granted from the admin panel itself and also from the kubectl by
+The Admin Panel is a section where the Kubermatic administrators can manage the global settings that
+impact all Kubermatic users directly. Admin rights can be granted from the admin panel itself and also from the kubectl by
 setting the `spec.admin` field of the user object to `true`.
 
 ```bash
 $ kubectl get user -o=custom-columns=INTERNAL_NAME:.metadata.name,NAME:.spec.name,EMAIL:.spec.email,ADMIN:.spec.admin
 $ kubectl edit user ...
 ```
 
-After logging in to the dashboard with an administrator you should be able to access the admin panel from the menu up
+After logging in to the dashboard as an administrator, you should be able to access the admin panel from the menu up
 top.
 
 ![](/img/kubermatic/master/ui/admin_panel_access.png?height=300px&classes=shadow,border "Accessing the Admin Panel")
 
 ![](/img/kubermatic/master/ui/panel.png?height=350px&classes=shadow,border "Admin Panel")
 
-The Admin Panel offers a wide set of features including but not limited to:
+## Global Settings
 
-## [Manage Custom Links]({{< ref "./custom_links" >}})
+This section explains how to manage KKP global settings such as custom links, dynamic datacenters among others.
+It can be done via UI using admin panel or from the command line with `kubectl`.
+
+### Changing the Global Settings Using `kubectl`
+
+Global settings are stored in `KubermaticSetting` custom resource named `globalsettings`.
+
+The resource has the following structure:
+
+```
+apiVersion: kubermatic.k8s.io/v1
+kind: KubermaticSetting
+metadata:
+  name: globalsettings
+  ...
+spec:
+  cleanupOptions:
+    Enabled: true
+    Enforced: false
+  clusterTypeOptions: 0
+  customLinks:
+  - icon: ""
+    label: Twitter
+    location: footer
+    url: https://www.twitter.com/kubermatic
+  - icon: ""
+    label: GitHub
+    location: footer
+    url: https://github.com/kubermatic
+  - icon: ""
+    label: Slack
+    location: footer
+    url: http://slack.kubermatic.io/
+  defaultNodeCount: 1
+  displayAPIDocs: true
+  displayDemoInfo: false
+  displayTermsOfService: true
+  enableDashboard: true
+  enableExternalClusterImport: true
+  enableOIDCKubeconfig: false
+  machineDeploymentVMResourceQuota:
+    enableGPU: true
+    maxCPU: 8
+    maxRAM: 64
+    minCPU: 1
+    minRAM: 2
+  restrictProjectCreation: false
+  userProjectsLimit: 0
+
+```
+
+It can be edited directly from the command line:
+
+```
+$ kubectl edit kubermaticsetting globalsettings
+```
+
+**Note:** Custom link `icon` is not required and defaults will be used if field is not specified. `icon` URL can
+point to the images inside the container as well, i.e. `/assets/images/icons/custom/github.svg`.
+
+## Manage Global Settings Via UI
```

Comment:
This would not be needed if we reorganize the order/remove the global settings section.